model_bakery/__about__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "1.9.0"
+__version__ = "1.9.1"
```

Comment:
This should only be updated by a maintainer when publishing a new release. It does not need to be altered by contributors, especially not during PR time. Please revert (force push to avoid polluting the git history).